.travis.yml

```diff
@@ -39,8 +39,12 @@ before_install:
       export GCC_COLORS='error=01;31:warning=01;35:note=01;36:caret=01;32:locus=01:quote=01';
     fi;
   # Add coverall for C++ so coverall.io could be triggered. Even it should be --coverage and gcov.
+  # Install updated libglew-dev since the version provided by trusty is outdated
```

Comment:
Does this mean that we need at least oGL 4.5 GPUs for this renderer? Kinda defeats the purpose since they should also support Vulkan.